package.json

```diff
@@ -9,8 +9,8 @@
   },
   "license": "MIT",
   "engines": {
-    "node": ">=16.15.1",
-    "npm": ">=8.11.1"
+    "node": ">=20.6.1",
+    "npm": ">=9.8.1"
```

Comment:
Morning Asif!

Whenever you notice npm listed under engines, then this minimum version got explicitely added because of security concerns.

This overview shows the nodejs to npm version matrix:
https://nodejs.org/en/download/releases

There, we see that they ship at least since Node.js 19.9.0 with a higher npm version than we've defined as minimum.
Means, we can basically remove the npm listing from the engines section.
20.6.1 is shipping with npm

Reply:
Good morning Kai, thanks for bringing that into my attention.
I will remove npm entries from all repos where we have explicitly defined in the package.json.